kindtest.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+set -o errexit
+
+if [[ -z "$WORKSPACE" ]]; then
+  kinddir="/scratch/$USER/kindtest"
```

Comment:
maybe we could print a usage message and stop, rather than arbitrarily chose this location?

Reply:
I was trying to have a useful default when running on your own machine.  I think that is still valuable.  I can make another argument.